flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/BoundedDataTestBase.java

```diff
@@ -91,10 +112,10 @@ public void testWriteAndReadDataAcrossRegions() throws Exception {
 
 	private void testWriteAndReadData(BoundedData bd) throws Exception {
 		final int numInts = 10_000_000;
-		final int numBuffers = writeInts(bd, numInts);
+		final int numBuffers = writeInts(bd, numInts, compressedEnabled);
```

Comment:
We can use `compressedEnabled` directly inside below `writeInts` method if this variable is static, then do not need to change this.